docs/concepts/primer.md

```diff
@@ -32,6 +33,25 @@ As any mature and advanced application development framework, .NET has many powe
 *   [Native interoperability](#native-interoperability)
 *   [Unsafe Code](#unsafe-code)
 
+## Programming languages
+
+As a developer, you can choose any programming language that supports .NET to create your application. Because .NET provides language independence and interoperability, you can interact with other .NET applications and components regardless of the language with which they were developed.
+
+Languages that allow you to develop applications for the .NET Platform adhere to the [Common Language Infrastructure (CLI) specification](https://www.visualstudio.com/en-us/mt639507).
+
+Microsoft languages that .NET supports include C#, F#, and Visual Basic. 
+
+* C# is simple, powerful, type-safe, and object-oriented while retaining the expressiveness and elegance of C-style languages. Anyone familiar with C and similar languages will find few problems in adapting to C#.
+
+* F# is a cross-platform, functional-first programming language that also supports traditional object-oriented and imperative programming.
+
+* Visual Basic is a productive, type-safe and object-oriented language that you can use to build a variety of applications that run on .NET.
+
+> **Note**
+> 
+> In the current RC2 release of .NET Core, only C# is fully supported.
```

Comment:
@stevehoag I believe there's going to be a branch for RTW?  We could remove the note in that branch and I'll remove the word "current" now.